sky/serve/autoscalers.py

```diff
@@ -16,32 +17,36 @@
 
 logger = sky_logging.init_logger(__name__)
 
-_UPSCALE_DELAY_S = 300
-_DOWNSCALE_DELAY_S = 1200
-
 
 class AutoscalerDecisionOperator(enum.Enum):
     SCALE_UP = 'scale_up'
     SCALE_DOWN = 'scale_down'
-    NO_OP = 'no_op'
 
 
 @dataclasses.dataclass
 class AutoscalerDecision:
     """Autoscaling decisions.
 
-    |---------------------------------------------------------|
-    | Operator   | TargetType | Meaning                       |
-    |------------|------------|-------------------------------|
-    | SCALE_UP   | int        | Number of replicas to add     |
-    |------------|------------|-------------------------------|
-    | SCALE_DOWN | List[int]  | List of replica ids to remove |
-    |------------|------------|-------------------------------|
-    | NO_OP      | None       | No scaling needed             |
-    |---------------------------------------------------------|
+    |---------------------------------------------------------------|
+    | Operator   | TargetType       | Meaning                       |
+    |------------|------------------|-------------------------------|
+    | SCALE_UP   | Dict[str, Any]   | Resource override to add      |
```

Comment:
This should be `Optional[Dict[str, Any]` since the resources override might be None?